microsoft_outlook_single_tenant/readme/USAGE.rst

```diff
@@ -0,0 +1,7 @@
+* Register a new application in azure
+* Set who can use this application or access this API as single-tenant
+* Set Outlook credentials Client ID and Client Secret in Odoo settings
+* Set a ir_config_parameter like that:
+
+[ir.config_parameter]
+microsoft_outlook_directory_tenant_id = Directory (tenant) ID
```

Comment:
```suggestion
* Register a new application on Azure
* Set who can use this application or access this API as single-tenant
* Set Outlook credentials Client ID and Client Secret in Odoo settings  -> where?
* Set a ir_config_parameter like that:

[ir.config_parameter]
microsoft_outlook_directory_tenant_id = Directory (tenant) ID
```
This should be `CONFIGURE.rst` and not `USAGE.rst`

Reply:
Done